API_request_test.py

```diff
@@ -3,6 +3,40 @@
 
 import requests
 import json
+import os, sys
+import argparse
+
+parser = argparse.ArgumentParser(
+                    prog='BetterWeather',
+                    description='This program is written in Python. Its purpose is to collect weather forecast data\n'
+                    'from a few websites (e.g. Open-Meteo, WeatherAPI) and combine it\n'
+                    'into one more accurate weather prediction. After providing the location eg. "Madrid"\n'
+                    'you will receive a file with weather data and a user-friendly plot.')
+
+# positional arguments
+parser.add_argument('location', metavar='location', type= str,
+                    help='enter location for which you want to get weather forecast e.g. Cracovia')
+
+# if no arguments were given, printing the help message (args = "--help")
+# args = parser.parse_args(args=None if sys.argv[1:] else ['--help'])
+args = parser.parse_args()
+print(args.location)
+
+keys_path = './better_weather_keys.txt'
+
+# Check if better_weather_keys.txt file exists and it is not empty
+if os.path.isfile(keys_path) and os.stat(keys_path).st_size != 0:
+    keys_file = open(keys_path, "r")
+else:
+    print("Warning: File better_weather_keys.txt is not in the working directory or it is empty!")
```

Comment:
```suggestion
    print("Error: File better_weather_keys.txt is not in the working directory or it is empty!")
    exit()
```
I believe it does not make sense to proceed with the execution when there are no keys, no keys -> no requests -> no results.

Reply:
As we discussed it, I agree.